### PR TITLE
Bot pr arch ha011ae

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Fix #2

File: conda-forge.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 compiler_stack: comp7
+conda_forge_output_validation: true
 max_py_ver: '37'
 max_r_ver: '35'
-provider:
-  win: azure
-conda_forge_output_validation: true
+provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
```
Comment (hmaarrfk) [inline conda-forge.yml]:
```suggestion
provider: {linux_aarch64: default, linux_ppc64le: default}
```

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Hi @hmaarrfk, do you think it's okay to specify two different automake versions, one for win and one for unix? I cannot see an easy way around it. If you're happy with the split, I think this PR is ready to be merged :)

File: recipe/build.sh [hunk 2]
```diff
@@ -45,7 +45,11 @@ else
       -I "$mprefix/share/aclocal"
   )
 fi
-am_version=1.15 # keep sync'ed with meta.yaml
+if [ -n "$CYGWIN_PREFIX" ] ; then
+  am_version=1.15 # keep sync'ed with am_version_win in meta.yaml
+else
+  am_version=1.16 # keep sync'ed with am_version in meta.yaml
```
Comment (hmaarrfk) [inline recipe/build.sh:51]: do we even need this.

**Comment by hmaarrfk** [comment]
Thanks!

**Pull request merged**